### ISSUE TITLE
Shouldn't here be a +=?

### ISSUE THREAD

**Issue by wendlerc** [DSPsleeporg/smiles-transformer#20]
https://github.com/DSPsleeporg/smiles-transformer/blob/7ffa26e5bc68db0292164466398ff2edfa4a0f62/smiles_transformer/pretrain_trfm.py#L58

**Comment by shionhonda** [comment]
~You're right. It should be `+=`~

**Comment by sevencheung2021** [comment]
> 

so ,line 58 should be a += ?

**Comment by wendlerc** [comment]
No I think at some point I noticed it is fine the way it is.

**Comment by VincentBt** [comment]
I agree with @chrislybaer, it is fine the way it is: there is no need for a "+="
See https://pytorch.org/tutorials/beginner/transformer_tutorial.html#define-the-model

**Comment by shionhonda** [comment]
@VincentBt You are definitely right. I'm sorry for confusing you all with my previous comment.
